### PR TITLE
[ICP-13333] Additional metering configuration for Dawon devices

### PR DESCRIPTION
**Pull request by PKacprowiczS**
@tpmanley @greens @KKlimczukS @MGoralczykS @ZWozniakS @MWierzbinskaS @BJanuszS

### CODE REVIEW

File: devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy [hunk 1]
```diff
@@ -150,7 +150,18 @@ def configure() {
     sendEvent(name: "checkInterval", value: 2 * 60 + 10 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
     log.debug "Configuring Reporting"
     return refresh() +
-    	   zigbee.onOffConfig() +
-           zigbee.configureReporting(zigbee.SIMPLE_METERING_CLUSTER, ATTRIBUTE_READING_INFO_SET, DataType.UINT48, 1, 600, 1) +
-           zigbee.electricMeasurementPowerConfig(1, 600, 1) 
+            zigbee.onOffConfig() +
+            zigbee.configureReporting(zigbee.SIMPLE_METERING_CLUSTER, ATTRIBUTE_READING_INFO_SET, DataType.UINT48, 1, 600, 1) +
+            zigbee.electricMeasurementPowerConfig(1, 600, 1) +
+            additionalConfig
```
Comment (greens) [inline devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy]: Won't this append `null` for non-Dawon devices? I don't know whether that will cause problems upstream or not.

Comment (greens) [reply devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy]: unless there's an example of us doing something similar elsewhere in the codebase

Comment (PKacprowiczS) [reply devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy]: Honestly, I'm not sure. I guess it'll be easier for me to just do it "sure way".

**Review by greens — DISMISSED** [review]

**Comment by greens** [comment]
looks good, just waiting for confirmation

**Comment by greens** [comment]
@PKacprowiczS you've got a conflict now too

**Comment by PKacprowiczS** [comment]
It looks like the same configuration has been added for all devices in another PR, #38788, so I'll close this one

**Pull request closed without merging**